app/models/record_formats/dif10_record.rb

```diff
@@ -59,24 +59,38 @@ def create_script(raw_data = nil)
       add_script_comment(comment_hash)
     end
 
+    def get_raw_concept(concept_id, format)
+      url = "#{Cmr.get_cmr_base_url}/search/concepts/#{concept_id}.#{format}"
+      raw_data = Cmr.cmr_request(url).parsed_response
+      raw_data
+    end
 
     # There are currently only scripts for DIF10 collections.
     def evaluate_script(raw_data = nil)
       if raw_data.nil?
```

Comment:
I remove the "if" statement here.   just have line 71, just in case some caller passes you something different.   we can refactor this later once we switch completely over to pyquarc